CourseApp/Program.cs

```diff
@@ -4,11 +4,43 @@ namespace CourseApp
 {
     public class Program
     {
+        public static void TaskA(double a, double b)
+        {
+            double y;
+            double numerator;
+            double denominator;
+            Console.WriteLine("------- TASK A -------");
```

Comment:
This is not the job of the TaskA - it should to make some outputs to the console